tools/PAMPA/PAMPA_GLM_SP.xml

```diff
@@ -0,0 +1,146 @@
+<tool id="pampa_glmsp" name="Compute GLM on population data" version="@VERSION@">
+    <description>Compute a GLM of your choice on population data</description>
+    <macros>
+        <import>pampa_macros.xml</import>
+    </macros>
+    <expand macro="GLM_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript 
+         '$__tool_directory__/FunctExeCalcGLMSpGalaxy.r' 
+         '$input_metric'
+         '$input_unitobs'
+         '$varint'
+         '$varrep'
+         '$varrand'
+         'species.code'
+         #if $settings.advanced=='advanced' 
+             $settings.distrib
+             $settings.log
+         #else
+             'None'
+             'FALSE'
+         #end if
+         'unitSp'
+         '$__tool_directory__/FunctPAMPAGalaxy.r' 
+         '$output_summary'
+         '$output_recap'
+    ]]>
+    </command>
+    <inputs>
+        <param name="input_metric" type="data" format="tabular" label="Input population metrics file" help="Metrics data file, with location, year, and metrics informations as 'presence absence', 'abundance' or others who can be used as interest variable."/>
+        <param name="input_unitobs" type="data" format="tabular" label="Unitobs informations file" help="Unitobs file, with all informations available about unitobs."/>
+        <param name="varint" type="data_column" data_ref="input_metric" label="Interest variable from population metrics file" help= "Choose the field of the interest variable."/>
+        <param name="varrep" type="select" label="Response variables" help= "Choose the response variables you want to include in your analysis." multiple="true">
```

Comment:
that could also be in the macros, most of this tool is shared with the other tool above.